src/full_node/weight_proof.py

```diff
@@ -300,9 +300,9 @@ async def __first_sub_slot_vdfs(
                 sub_slots_num = 2
                 while sub_slots_num > 0 and curr_sub_rec.height > 0:
                     curr_sub_rec = blocks[curr_sub_rec.prev_hash]
-                if curr_sub_rec.first_in_sub_slot:
-                    assert curr_sub_rec.finished_challenge_slot_hashes is not None
-                    sub_slots_num -= len(curr_sub_rec.finished_challenge_slot_hashes)
+                    if curr_sub_rec.first_in_sub_slot:
```

Comment:
Can you describe the reason for this change in the PR?

Reply:
sure, added